<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPBYMINW1906" tests="3" failures="2" timestamp="24 Apr 2015 08:44:34 GMT" time="0.028" errors="0">
  <testcase name="COMPARE_BY_DESC_TYPE_Test" time="0.011" classname="ServicesTest"/>
  <testcase name="readAllExceptionTest" time="0.002" classname="ServicesTest">
    <failure type="org.testng.TestException" message="
Method ServicesTest.readAllExceptionTest()[pri:0, instance:ServicesTest@9fc1696] should have thrown an exception of type class java.lang.IndexOutOfBoundsException">
      <![CDATA[org.testng.TestException: 
Method ServicesTest.readAllExceptionTest()[pri:0, instance:ServicesTest@9fc1696] should have thrown an exception of type class java.lang.IndexOutOfBoundsException
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- readAllExceptionTest -->
  <testcase name="readAllTest" time="0.0" classname="ServicesTest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:635)
	at java.util.ArrayList.get(ArrayList.java:411)
	at ServicesTest.readAllTest(ServicesTest.java:21)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- readAllTest -->
</testsuite>
